geth/whisper/mailservice.go

```diff
@@ -0,0 +1,46 @@
+package whisper
+
+import (
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// MailService is a service that provides some additional Whisper API.
+type MailService struct {
+	nodeManager common.NodeManager
```

Comment:
We should be able to change it to `node.Node`.

Reply:
If we won't to add an interface keep `nodeManager` for futher refactoring is ok for me.